testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/jackson2/resource/Jackson2XmlResource.java

```diff
@@ -13,7 +13,7 @@
    @GET
    @Produces("application/json")
    @Path("{id}")
-   @BadgerFish
+//   @BadgerFish
```

Comment:
Can we simply remove the badgerfish annotation ?

Reply:
yes we can. Jackson's JSON output is okay.